test/SILGen/objc_access_notes.swift

```diff
@@ -551,6 +551,15 @@ func useWotsit(_ w: Wotsit<String>) {
 
 func other() { }
 
+// expected-remark@+2 {{implicitly added '@usableFromInlin' to this struct, as specified by access note for fancy test suite}}
+// expected-note@+1 {{add '@usableFromInlin' explicitly to silence this warning}} {{1-1=@usableFromInline}}
```

Comment:
The attribute name in these diagnostics is misspelled—if they pass, something must be wrong elsewhere.

Reply:
Yup, there is indeed something wrong. I'll investigate

Reply:
Fixed. The existing logic for pretty printing the attributes for diagnostics assumed too much about the output of `ASTPrinter`.